library/api/library.api

```diff
@@ -0,0 +1,55 @@
+public final class com/chuckerteam/chucker/BuildConfig {
+	public static final field BUILD_TYPE Ljava/lang/String;
```

Comment:
Same comment about `BuildConfig` as above.

Reply:
As discussed, let's do this in a separate PR 👌